style.css

```diff
@@ -294,16 +296,16 @@
 /* MAIN FEED */
 
 /* GENERIC POST ADJUSTMENTS */
-.main-feed {
-    /* overflow-y: scroll; */
-}
 
 .main-feed > div {
     display: grid;
     grid-template-columns: 50px 1fr;
     grid-auto-rows: auto;
     padding: 15px 15px;
     border-bottom: solid 1px var(--border-color);
+    border-left: solid  1px var(--border-color);
+    border-right: solid  1px var(--border-color);
```

Comment:
Quite curious about the var(--border-color) bit - first time I've stumbled upon that! Mind explaining what its job is? Cheers!

Reply:
So these are just global variables I've created. If you see at the top of my CSS the ' :root ' part is where I've made a few variables (just colours for this example) that I know I will be using multiple times throughout my code. You can then just refer back to that variable you created using var() later in the css, which is helpful if you create a specific style, colour, etc that may be unique. Hope this helps!